main/windows/main.js

```diff
@@ -102,6 +103,26 @@ function setAspectRatio (aspectRatio) {
   main.win.setAspectRatio(aspectRatio)
 }
 
+function setContentSize (size) {
```

Comment:
Looks like we still need `setBounds` to restore window size and position after exiting from a video, but we can probably simplify it. It no longer needs to do autocentering, where `x` and `y` are optional arguments.